successes.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: singlepage
+description: "Success Stories and Testimonials"
+author: jeff
+---
+
+
+<div markdown="1" class="container">
+
+  <h2 class="section-heading text-center">Recent Successes</h2>
+  <h3 class="section-subheading text-center text-muted">Twitter</h3>
+
+  * Developed a custom Incremental Change (IC) solution for releasing Puppet code changes on a schedule across failure domains.
```

Comment:
The raw bulleted list that these get rendered as feels a bit too... texty.

Maybe try this, which uses markdown headers that get rendered as HTML headers, and I think looks a bit more natural on the page:

```html
<div markdown="1" class="container">                                                                                                            
<h2 class="section-heading text-center">Recent Successes</h2>                                                                                   
                                                                                                                                                
## Twitter                                                                                                                                      
                                                                                                                                                
### Incremental change Puppet plugin and workflow                                                                                               
                                                                                                                                                
  Developed a custom incremental change (IC) solution for releasing Puppet code changes on a schedule across failure domains.                   
  * IC functions as a feature flag workflow that allows for pausing or rolling back individual Puppet code changes based on configuration flags.
```

Note that I re-worded the first bullet item to read more as a section description than a list item.

Reply:
Synced with Nate on this, which resulted in the most recent commit.